clients/drcachesim/simulator/tlb.h

```diff
@@ -46,21 +49,67 @@ namespace drmemtrace {
 
 class tlb_t : public caching_device_t {
 public:
+    tlb_t(const std::string &name = "tlb");
+
     void
     request(const memref_t &memref) override;
 
-    // TODO i#4816: The addition of the pid as a lookup parameter beyond just the tag
-    // needs to be imposed on the parent methods invalidate(), contains_tag(), and
-    // propagate_eviction() by overriding them.
+    // TODO i#4816: The addition of the pid as a lookup parameter beyond just the
+    // tag needs to be imposed on the parent methods invalidate(), contains_tag(),
+    // and propagate_eviction() by overriding them.
+
+    // Returns the replacement policy used by this TLB.
+    virtual std::string
+    get_replace_policy() const override = 0;
```

Comment:
This is already present in the parent class as LFU: please add a comment explaining this and why this is overriding it.  Probably include a `XXX i#7287: ...` comment on refactoring replacement policies.